akka-stream/src/main/scala/akka/stream/javadsl/Source.scala

```diff
@@ -683,7 +687,7 @@ final class Source[Out, Mat](delegate: scaladsl.Source[Out, Mat]) extends Graph[
    * Connect this `Source` to a `Sink` and run it. The returned value is the materialized value
    * of the `Sink`, e.g. the `Publisher` of a `Sink.asPublisher`.
    *
-   * Prefer the method taking an ActorSystem unless you have special requirements.
+   * Prefer the method taking an `ActorSystemProvider` unless you have special requirements.
```

Comment:
it's ClassicActorSystemProvider,  we can write `ActorSystem` here since that is what the parameter is in practice